ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -563,12 +562,12 @@ sub set_Title_from_podtree {
             (@$podtree == 4 ||
                 (ref($podtree->[4]) eq "ARRAY" &&
                 $podtree->[4]->[0] eq "head1"))) {
-            $globals->{Title} = join("", @{$podtree->[3]}[2..$#{$podtree->[3]}]);
+            $self->{Title} = join("", @{$podtree->[3]}[2..$#{$podtree->[3]}]);
```

Comment:
prime candidate to be postfix

```
join("", $podtree->[3]->@[ 2 .. $podtree->[3]->$#* ]);
```

Or, what I'd actually write:
```
my @want_indices = (2 .. $podtree->[3]->$#*);
… = join("", $podtree->[3]->@[ $want_indices ]);
```

I have refrained from a lot of "but postfix deref!" but in this case it seems to be to be a significant improvement — but only in clarify, not correctness.

Reply:
> prime candidate to be postfix
> 
> ```
> join("", $podtree->[3]->@[ 2 .. $podtree->[3]->$#* ]);
> ```
> 
> Or, what I'd actually write:
> 
> ```
> my @want_indices = (2 .. $podtree->[3]->$#*);
> … = join("", $podtree->[3]->@[ $want_indices ]);
> ```
> 
> I have refrained from a lot of "but postfix deref!" but in this case it seems to be to be a significant improvement — but only in clarify, not correctness.

In keeping with the approach I've been taking throughout this project, I'd like to defer introducing postfix deref (where appropriate) until after we've gotten the OO-ified Pod::Html in blead.  That way, if using that proves at all problematic, we will have isolated the problems.  I will add this to the TODO.